joomla/joomla-cms
# [4.0] Namespace com_tags

Pull Request for Issue # .

### Summary of Changes
This PR converts com_tags component to namespace MVC for Joomla 4.

### Testing Instructions
As the none namespace version has issues, the test is just makes sure it works as before:

1. Backend: Test and make sure add/edit/publish/unpublish.. works as expected.
2. Frontend: Currently, view detail tag of some tags in sample data broken (routing issue, I guess). So just view list of tags, view some tags work

### Known issues

1. Routing to tag detail broken (for example Green tag in sample data)

2. Viewing rss feed has some notices:

- Notice: Undefined index: mime in E:\www\contact\libraries\joomla\document\feed.php on line 217
- Notice: Undefined variable: tz in E:\www\contact\libraries\joomla\document\renderer\feed\rss.php on 247
- Warning: DateTime::setTimezone() expects parameter 1 to be DateTimeZone, null given in E:\www\contact\libraries\joomla\date\date.php on line 408
